test/system/roles/enroller/enroller_dashboard_test.rb

```diff
@@ -18,10 +18,6 @@ class EnrollerDashboardTest < ApplicationSystemTestCase
     @@enroller_test_helper.view_enrolled_monitorees('locals2c4_enroller')
   end
 
-  test 'view enrollment analytics' do
```

Comment:
Why was this removed?

Reply:
the EnrollerAnalytics component was no longer being used which is why this test was failing.  I went ahead and removed the related tests and components but it appears that the components are still here and being used?  talking to @DPC15038 about this rn

Reply:
I'm going to add these back in and see what's happening

Reply:
added back in and tests are fixed